pkg/virt-handler/rest/console.go

```diff
@@ -133,12 +133,6 @@ func (t *ConsoleHandler) getUnixSocketPath(vmi *v1.VirtualMachineInstance, socke
 	if _, err = os.Stat(socketPath); os.IsNotExist(err) {
 		return "", err
 	}
-	// This is a workaround preventing QEMU from deleting its sockets prematurely as described in a bug https://bugs.launchpad.net/qemu/+bug/1795100
-	// once the QEMU 4.0 is released the need for this workaround goes away
-	// Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1683964
-	if err = os.Chmod(socketDir, 0444); err != nil {
```

Comment:
I'm hesitant to remove this. Are we certain there's no environment which uses an impacted qemu still?

Reply:
If I'm not mistaken, the QEMU that we use is from the copr at [virtmaint-sig](https://copr.fedorainfracloud.org/groups/g/virtmaint-sig/coprs/), considering that Fedora 30 hit EOL and that was the only one with QEMU < 4.0 (without the fix), I thought that this was a safe code removal.

But I'm not confident on how the upgrading process works. Would it be possible that we have virt-handler upgraded to v0.33 while we have a QEMU running from KubeVirt < 0.24 ?

Reply:
likely not

Still, unless this is causing an issue, i wouldn't touch it. I don't think the risk/reward here is in our favor. The original problem this fixed was kind of a nightmare. VNC looked like it worked, then it magically wouldn't... but it would look like a strange routing issue.

Reply:
I'll remove the patch for now, no problem. Should we consider when it is sane to either remove this code or not to mark it as workaround?